specification/monitor/resource-manager/Microsoft.Insights/preview/2023-06-01-preview/privateLinkScopes_API.json

```diff
@@ -0,0 +1,1167 @@
+        },
+        "ingestionAccessMode": {
+          "description": "Specifies the access mode of ingestion through the specified private endpoint connection in the exclusion.",
+          "$ref": "#/definitions/AccessMode"
+        }
+      }
+    },
+    "AccessMode": {
+      "description": "Access mode types.",
+      "type": "string",
+      "enum": [
+        "Open",
+        "PrivateOnly"
+      ],
+      "x-ms-enum": {
+        "name": "AccessMode",
+        "modelAsString": true
+      }
+    },
+    "AzureMonitorPrivateLinkScopeListResult": {
+      "description": "Describes the list of Azure Monitor PrivateLinkScope resources.",
+      "type": "object",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of Azure Monitor PrivateLinkScope definitions.",
+          "items": {
+            "$ref": "#/definitions/AzureMonitorPrivateLinkScope"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to get the next set of Azure Monitor PrivateLinkScope definitions if too many PrivateLinkScopes where returned in the result set."
+        }
+      }
+    },
+    "ScopedResourceListResult": {
+      "description": "A list of scoped resources in a private link scope.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScopedResource"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ScopedResource": {
+      "description": "A private link scoped resource",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ScopedResourceProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "description": "System data",
+          "readOnly": true
+        }
+      }
+    },
+    "ScopedResourceProperties": {
+      "description": "Properties of a private link scoped resource.",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "The kind of scoped Azure monitor resource.",
+          "type": "string",
+          "readonly": "true",
+          "enum": [
+            "Resource",
+            "Metrics"
+          ],
+          "x-ms-enum": {
+            "name": "ScopedResourceProvisioningState",
```

Comment:
>ScopedResourceProvisioningState [](http://example.com/codeflow?start=21&length=31)

Must not have duplicate name of x-ms-enum extension , make sure every x-ms-enum name unique.

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/unique-xms-enum-name.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/unique-xms-enum-name.md)

Reply:
please resolve this comment and follow [Swagger-Suppression-Process](https://aka.ms/azsdk/pr-suppressions) to suppress [ResourceNameRestriction](https://github.com/Azure/azure-openapi-validator/blob/main/docs/resource-name-restriction.md) in https://github.com/Azure/azure-rest-api-specs/pull/24590/checks?check_run_id=17587599023 with breaking change reason. After that, please remove **ARMChangesRequested** label for another review.